### PR TITLE
Added API integration

### PR DESCRIPTION
**Pull request by JISevilla92**
## Summary
- Login, dashboard and book details are all now connected with the API

### CODE REVIEW

File: src/components/customComponents/apiService.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const ApiInstance = axios.create({
+    baseURL:'https://wbooks-api-stage.herokuapp.com/api/v1/'
+});
+
+
+const ApiService = {
```
Comment (sbalay) [inline src/components/customComponents/apiService.js]: create different services for each business unit. here we should have an AuthService and a BookService

File: src/components/authentication/privateRoute.js [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
   render() {
     const logged = localStorage.getItem('loggedUser') ? true : false
+    const Comp = this.props.component;
     return (
       <Route render={props => (
-        logged ? React.createElement(
-          this.props.component,
-          {...this.props}
-        ) : (
+        logged ? ApiService.resetToken() && <Comp {...this.props} />
```
Comment (sbalay) [inline src/components/authentication/privateRoute.js]: shouldn't this be: `logged && ApiService.resetToken() ?`

File: src/components/authentication/privateRoute.js [hunk 3]
```diff
@@ -1,16 +1,15 @@
 import React, { Component } from 'react';
 import { Route, Redirect } from 'react-router-dom'
-
+import ApiService from '../customComponents/apiService';
 class PrivateRoute extends Component {
 
   render() {
     const logged = localStorage.getItem('loggedUser') ? true : false
```
Comment (sbalay) [inline src/components/authentication/privateRoute.js]: the ternary operator here is unnecessary: ` const logged = localStorage.getItem('loggedUser');`

**Review by sbalay — CHANGES_REQUESTED** [review]

File: src/components/authentication/privateRoute.js (hunk 2 above)
Comment (JISevilla92) [reply src/components/authentication/privateRoute.js]: I think it could be either way, but the one you say seems more readable

Comment (JISevilla92) [reply src/components/authentication/privateRoute.js]: ended up removing this and doing differently

**Pull request closed without merging**